service/src/main/java/bio/terra/workspace/db/WorkspaceDao.java

```diff
@@ -292,24 +414,26 @@ public void deleteCloudContext(UUID workspaceId, CloudPlatform cloudPlatform) {
    *
    * @param workspaceId workspace holding the context
    * @param cloudPlatform platform of the context
-   * @param flightId if non-null, then it is compared to the creating_flight to make sure a
-   *     conflicting create does not delete a valid cloud context
+   * @param creatingFlightId if non-null, then it is compared to the creating_flight to make sure a
+   *     conflicting create does not delete a valid cloud context. This behavior can be removed when
+   *     we are able to do PF-1238.
    */
   private void deleteCloudContextWorker(
-      UUID workspaceId, CloudPlatform cloudPlatform, @Nullable String flightId) {
+      UUID workspaceId, CloudPlatform cloudPlatform, @Nullable String creatingFlightId) {
     final String platform = cloudPlatform.toSql();
     String sql =
         "DELETE FROM cloud_context "
-            + "WHERE workspace_id = :workspace_id AND cloud_platform = :cloud_platform";
+            + "WHERE workspace_id = :workspace_id"
+            + " AND cloud_platform = :cloud_platform";
 
     MapSqlParameterSource params =
         new MapSqlParameterSource()
             .addValue("workspace_id", workspaceId.toString())
             .addValue("cloud_platform", platform);
 
-    if (StringUtils.isNotEmpty(flightId)) {
-      sql = sql + " AND creating_flight = :flightid";
-      params.addValue("flightid", flightId);
+    if (StringUtils.isNotEmpty(creatingFlightId)) {
+      sql = sql + " AND creating_flight = :creating_flight";
+      params.addValue("creating_flight", creatingFlightId);
```

Comment:
creating_flight_id?

Reply:
The name "creating_flight" is in the schema and would require an upgrade to rename.